extensions/ql-vscode/src/config.ts

```diff
@@ -715,6 +715,7 @@ const LLM_GENERATION_DEV_ENDPOINT = new Setting(
   "llmGenerationDevEndpoint",
   MODEL_SETTING,
 );
+const MODEL_EVALUATION = new Setting("evaluation", MODEL_SETTING);
```

Comment:
Should this go in our internal feature flag list? 🎏

Reply:
Yep, I'll update that.